components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/BrowserAction.kt

```diff
@@ -25,4 +25,21 @@ data class BrowserAction(
     val badgeTextColor: Int?,
     val badgeBackgroundColor: Int?,
     val onClick: () -> Unit
-)
+) {
+    /**
+     * Return a copy of [BrowserAction] with tab-specific [override] if it exists.
+     *
+     * @param override tab-specific action only contain the values that should be overridden.
```

Comment:
nit: `@param override the action to use for overriding properties. Note that only the provided (non-null) properties of the override will be applied, all other properties will remain unchanged.`